resources/widgets.sass

```diff
@@ -68,6 +72,10 @@ input
   &[type=number]
     padding: 2px 0 2px 5px
 
+.disabled
```

Comment:
Remove this and add !important to the `border-color` for the other one and I'll just merge it